### ISSUE TITLE
请问 现在支持caffe的哪些层？

### ISSUE THREAD

**Issue by ppl21847** [XiaoMi/mace#335]
如题，请问现在支持caffe的哪些层？谢谢。

**Comment by ppl21847** [comment]
self._op_converters = {
            'Input': self.convert_nop,
            'Convolution': self.convert_conv2d,
            'Deconvolution': self.convert_deconv2d,
            'Eltwise': self.convert_elementwise,
            'Add': self.convert_add,
            'ReLU': self.convert_activation,
            'TanH': self.convert_activation,
            'Sigmoid': self.convert_activation,
            'PReLU': self.convert_activation,
            'Pooling': self.convert_pooling,
            'Concat': self.convert_concat,
            'Slice': self.convert_slice,
            'Softmax': self.convert_softmax,
            'InnerProduct': self.convert_fully_connected,
            'BatchNorm': self.convert_folded_batchnorm,
            'Crop': self.convert_crop,
            'Scale': self.convert_scale,
            'ShuffleChannel': self.convert_channel_shuffle,
            'Permute': self.convert_permute,
            'Flatten': self.convert_flatten,
            'PriorBox': self.convert_prior_box,
            'Reshape': self.convert_reshape,
        }

这是有所支持caffe的层吗？

**Comment by nolanliou** [comment]
是的

**Comment by ppl21847** [comment]
>

> 是的

好的，谢谢。

**Comment by ppl21847** [comment]
> 是的

麻烦问一下 如何caffe自定义层？

**Comment by nolanliou** [comment]
可以看一下这个https://mace.readthedocs.io/en/latest/development/adding_a_new_op.html，有问题可以再交流

**Comment by ppl21847** [comment]
> 可以看一下这个文档https://mace.readthedocs.io/en/latest/development/adding_a_new_op.html
> 有问题可以再交流

好的，謝謝。
